### ISSUE TITLE
Usability: NVDA is not reciting any information after a Git Branch is changed.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17261]
VS Code Version: 1.6.1
OS Version:1607
NVDA version:-2016.2.1

**Steps to Reproduce:**
1.Open Visual studio code editor.
2. Open any file and using "Ctrl+1" key  navigate to the first vs code editor.
3. Using F1 key navigate to the palette, type "git branch" and "refactor-to-promise" as the branch name and Hit Enter and observe NVDA announcement.

**Expected result:**
NVDA should provide information after a Git Branch is changed.

**Actual Result:**
NVDA is not reciting any information after a Git Branch is changed
Note: NVDA Speech Viewer is added in the Screenshot

**User Impact:**
Screen Reader user doesn't get to know whether the Git Branch is changed or Not.

**Attachments:**
![nvda is not reciting any information after a git branch is changed](https://cloud.githubusercontent.com/assets/22956130/21215064/c6376cb2-c2c4-11e6-8a69-2d43f9f0188a.jpg)

**Comment by isidorn** [comment]
Similar as https://github.com/Microsoft/vscode/issues/17259 the issue is that Git is an extension and currently we have no story for extensions being able to announce aria statuses or to contribute in any way what is announced to NVDA. FYI @jrieken to consider introducing this in our extension API

However unlike #17259 it seems this functionality is very Git specific and I do not see a way of solving this on the vscode side.

**Comment by jrieken** [comment]
> FYI @jrieken to consider introducing this in our extension API

Unsure what API is required? An `alert`-call? In general, our API should be data'ish enough to bring us into the position in which can fill-in aria-labels et al. Or is this a problem that we need super special labels?

**Comment by isidorn** [comment]
@jrieken in this particular case we need an `alert` call and I think that would be a good starting point. As for the aria-labels that feels like we are exposing UI elements which is not in the spirit of our API

**Comment by isidorn** [comment]
Pushing this out to June (potentially Backlog) to discuss then how we can tackle this issue.
With the current extension API we can not fix this

**Comment by joaomoreno** [comment]
To keep the number of issues in our inbox on a manageable level, we're closing issues that have been on the backlog for a long time but haven't gained traction: We look at the number of votes the issue has received and the number of duplicates issues filed.

Thanks for your understanding. Happy coding!
